stable/rabbitmq/README.md

```diff
@@ -77,10 +77,18 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `rabbitmq.configuration`             | Required cluster configuration                   | See values.yaml                                         |
 | `rabbitmq.extraConfiguration`        | Extra configuration to add to rabbitmq.conf      | See values.yaml                                         |
 | `rabbitmq.advancedConfiguration`     | Extra configuration (in classic format) to add to advanced.config    | See values.yaml                                         |
+| `rabbitmq.ssl.enabled`                  | Enable ssl support to rabbitmq |                     |
+| `rabbitmq.ssl.failIfNoPeerCert`     | `true`  | When set to true, TLS connection will be rejected if client fails to provide a certificate |
+| `rabbitmq.ssl.sslOptionsVerify`     | `verify_peer`  | Should [peer verification](https://www.rabbitmq.com/ssl.html#peer-verification) be enabled? |
+| `rabbitmq.ssl.caCertificate`     | Ca certificate  | Certificate Authority (CA) bundle content |
+| `rabbitmq.ssl.serverCertificate`     | Server certificate  | Server certificate content |
+| `rabbitmq.ssl.serverKey`     | Server Key  | Server private key content |
 | `service.type`                       | Kubernetes Service type                          | `ClusterIP`                                             |
 | `service.port`                       | Amqp port                                        | `5672`                                                  |
+| `service.port_ssl`                   | Amqp ssl port                                    | `5671`                                                  |
 | `service.distPort`                   | Erlang distribution server port                  | `25672`                                                 |
 | `service.nodePort`                   | Node port override, if serviceType NodePort      | _random available between 30000-32767_                  |
+| `service.nodePort_ssl`               | Node port override, if serviceType NodePort      | _random available between 30000-32767_                  |
```

Comment:
Same here, can you replace it by something like
```suggestion
| `service.nodePortSSL`               | Node port override, if serviceType NodePort      | _random available between 30000-32767_                  |
```
?

Reply:
changed